cmd/create.go

```diff
@@ -20,8 +21,9 @@ import (
 
 func init() {
 	inletsCmd.AddCommand(createCmd)
-	createCmd.Flags().StringP("provider", "p", "digitalocean", "The cloud provider - digitalocean, packet, scaleway, or civo")
+	createCmd.Flags().StringP("provider", "p", "digitalocean", "The cloud provider - digitalocean, gce, packet, scaleway, or civo")
 	createCmd.Flags().StringP("region", "r", "lon1", "The region for your cloud provider")
+	createCmd.Flags().StringP("zone", "z", "us-central1-a", "The zone for the exit node (Google Compute Engine)")
```

Comment:
Wouldn't Europe as a default zone make more sense for us, or was there another reason on picking that zone in central US?

Reply:
Yes it would but the free 730hrs/month you get free for f1-micro only applies if the region is us-central1, else you get charged ~$4 (the price varies from 4.78 to 4.28 from europe to us)